sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/keyvault/certificates/properties/AzureKeyVaultCertificateProperties.java

```diff
@@ -8,18 +8,28 @@
 import com.azure.spring.service.keyvault.certificates.KeyVaultCertificateProperties;
 
 /**
- * Properties for Azure Key Vault Certificate.
+ * Azure Key Vault Certificate properties.
+ *
+ * @since 4.0.0
  */
-public class AzureKeyVaultCertificateProperties extends AzureKeyVaultProperties implements KeyVaultCertificateProperties {
+public final class AzureKeyVaultCertificateProperties extends AzureKeyVaultProperties implements KeyVaultCertificateProperties {
 
     public static final String PREFIX = "spring.cloud.azure.keyvault.certificate";
 
     private CertificateServiceVersion serviceVersion;
```

Comment:
also add java doc here

Reply:
This is a private field, no need to add java doc here. And here is no related requirement in api-review page.

Reply:
This is for configuration properties metada.